index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href=style.css>
```

Comment:
- [ ] [Optional] I am suggesting adding qoutes to the `href` attribute. You can refer to this reference for the [how to create a link](https://www.w3schools.com/html/html_links.asp)